tests/tensorflow/common.libsonnet

```diff
@@ -41,6 +41,49 @@ local volumes = import 'templates/volumes.libsonnet';
       },
     },
   },
+  TfNlpVisionMixin:: {
+    local config = self,
+    experiment: error 'Must define `experiment`',
+    additionalOverrides:: {},
```

Comment:
You don't need to add this field. Children can do something like this to override values in `paramsOverride`:

```jsonnet
{
  scriptConfig+: {
    paramsOverride+: {
      trainer: {
        train_steps: 10000,
        validation_interval: 10000,
      },
    }
  }
}
```

Use the `+:` syntax to override nested fields.

Reply:
Done